node-php/node8-php7.0/Dockerfile

```diff
@@ -23,6 +23,10 @@ RUN apt-get update && \
         php7.0-pdo-mysql \
     && apt-get clean
 
-WORKDIR /var/www/html
+RUN mkdir -p /home/app \
+    && usermod -d /home/app -l app node \
+    && chown -R app /home/app
 
-CMD tail -f /dev/null
\ No newline at end of file
+USER app
```

Comment:
If the default user is `app` it migth make sense to already set the default value of `NPM_CONFIG_PREFIX` in the Dockerfile too

- https://github.com/ModestCoders/magento2-dockergento/blob/master/docker-compose.yml#L65

Reply:
NPM_CONFIG_PREFIX is no longer needed because the default user in the is already app.